### PR TITLE
Update tutorials, closes MISC-212

### PR DESCRIPTION
**Pull request by tpetillot**
WIP IMAGES: gradle, sbt, bundle, includes

### CODE REVIEW

File: content/tutorials/_includes/introduction.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
-In this tutorial, we'll describe every step you need to take in order to have a working basic configuration on FrontLine.
+## Introduction
+
+In this tutorial, we'll describe every steps needed to run your fist Simulation with Gatling Enterprise.
```
Comment (Isammoc) [inline content/tutorials/_includes/introduction.md:3]:
```suggestion
In this tutorial, we'll describe every steps needed to run your first Simulation with Gatling Enterprise.
```

File: content/tutorials/_includes/introduction.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+Once connected, you'll have to create your own organization.
+
+{{< img src="organization-creation.png" alt="Organization Creation" caption="Organization Creation" >}}
+
+{{< alert warning >}}
+You'll be able to edit your organization name afterward, but not the slug.
+**Once in an organization, you cannot join another one for now.**
+{{< /alert >}}
+
+A new [organization]({{< ref "../../../reference/user/organization" >}}) is given **free credits** in order to **run your first simulation run**.
```
Comment (Isammoc) [inline content/tutorials/_includes/introduction.md:18]: Either:
 * run your first simulation
 * launch your first simulation run

File: content/tutorials/_includes/upload_api.md [hunk 3]
```diff
@@ -3,26 +3,25 @@ The option is also useful if you want to automate this step.
 
 First, you need to create an API Token as our REST API is secured.
 
-Back in FrontLine, click on the {{< icon key >}} icon on the Sidenav (navigation bar on the left of the screen).
+Back in FrontLine, click on **API Tokens** in the left navigation bar.
```
Comment (Isammoc) [inline content/tutorials/_includes/upload_api.md:6]:
```suggestion
Back in Gatling Enterprise Cloud, click on **API Tokens** in the left navigation bar.
```

File: content/tutorials/_includes/upload_api.md [hunk 4]
```diff
@@ -3,26 +3,25 @@ The option is also useful if you want to automate this step.
...
 
 {{< img src="api_token_navigation.png" alt="Sidenav navigation" >}}
 
-Click on the Create button in order to create an API Token in FrontLine.
+Click on the Create button in order to create an API Token.
```
Comment (Isammoc) [inline content/tutorials/_includes/upload_api.md:10]:
```suggestion
Click on the **Create** button in order to create an API Token.
```

File: content/tutorials/_includes/upload_manual.md [hunk 5]
```diff
@@ -11,3 +13,5 @@ You now have to upload the artifact you've generated earlier with Maven. Click o
 {{< img src="open_upload.png" alt="Open upload modal" >}}
 
 The artifact you've generated earlier is located here: `{param}`
+
+Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
```
Comment (Isammoc) [inline content/tutorials/_includes/upload_manual.md:17]:
```suggestion
Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
```

File: content/tutorials/bundle/index.md [hunk 6]
```diff
@@ -2,19 +2,21 @@
...
 weight: 50
 ---
 
 {{< include introduction.md >}}
 
-There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+## Download the zip bundle
 
-In this tutorial, we'll use the Gatling bundle. Make sure that you have a JDK 8 or 11 installed on your computer.
+There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run.
```
Comment (Isammoc) [inline content/tutorials/bundle/index.md:15]:
```suggestion
There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run. 
```

File: go.mod [hunk 7]
```diff
@@ -3,3 +3,4 @@ module github.com/gatling/frontline-cloud-doc
 go 1.16
 
 require github.com/gatling/gatling.io-doc-theme v0.0.0-20210804083439-31cf767b4028 // indirect
+replace github.com/gatling/gatling.io-doc-theme => /home/tpetillot/Desktop/workspace/gatling/repositories/gatling.io-doc-theme
```
Comment (Isammoc) [inline go.mod:6]: Remove that!

File: content/tutorials/_includes/resources.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 You have successfully started load testing, you can read our documentation if you need more information about our product:
 
-- [FrontLine documentation]({{< ref "../../" >}})
+- [Gatling Enterprise documentation]({{< ref "../../" >}})
 - [Documentation about the creation of an artifact]({{< ref "../../reference/user/reports" >}})
 - [Documentation about how to use the FrontLine reports]({{< ref "../../reference/user/reports" >}})
```
Comment (ccedric) [inline content/tutorials/_includes/resources.md:5]: ~~FrontLine~~

File: content/tutorials/_includes/upload_api.md [hunk 9]
```diff
@@ -3,26 +3,25 @@ The option is also useful if you want to automate this step.
```
Comment (ccedric) [inline content/tutorials/_includes/upload_api.md:3]: We should remove this whole section, which has nothing to do there. The goal of these tutorials is to have as few steps as possible to have a working configuration.

File: content/tutorials/_includes/upload_manual.md [hunk 10]
```diff
@@ -1,8 +1,10 @@
-Let's get back to FrontLine. To navigate to the artifacts page, click on the {{< icon archive >}} icon on the Sidenav (navigation bar on the left of the screen).
+Let's get back to Gatling Enterprise. 
+To navigate to the artifacts page, click on the **Artifacts** on the left navigation bar.
 
 {{< img src="artifact_navigation.png" alt="Sidenav navigation" >}}
 
-Click on the create button in order to create an artifact in FrontLine. Choose the name you want, then click save.
+Click on the create button in order to create an artifact. 
+Choose the name and the team you want, then click save.
```
Comment (ccedric) [inline content/tutorials/_includes/upload_manual.md:7]: no need to mention the team, keep it simple.

File: content/tutorials/_includes/introduction.md (hunk 1 above)
Comment (ccedric) [reply content/tutorials/_includes/introduction.md:3]: every step~~s~~

File: content/tutorials/_includes/introduction.md [hunk 11]
```diff
@@ -1,6 +1,21 @@
...
+It all begins on [cloud.gatling.io](https://cloud.gatling.io), where a GitHub account is needed in order to connect.
+
+{{< img src="login.png" alt="Login" caption="Login" >}}
+
+Once connected, you'll have to create your own organization.
+
+{{< img src="organization-creation.png" alt="Organization Creation" caption="Organization Creation" >}}
+
+{{< alert warning >}}
+You'll be able to edit your organization name afterward, but not the slug.
```
Comment (ccedric) [inline content/tutorials/_includes/introduction.md:14]: No need to mention that, keep it simple.

File: content/tutorials/bundle/index.md [hunk 12]
```diff
@@ -2,19 +2,21 @@
...
 {{< include introduction.md >}}
 
-There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+## Download the zip bundle
 
-In this tutorial, we'll use the Gatling bundle. Make sure that you have a JDK 8 or 11 installed on your computer.
+There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. 
+We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+
+In this tutorial, we'll use the **Gatling bundle**. Make sure that you have a JDK 8 or 11 installed on your computer.
```
Comment (ccedric) [inline content/tutorials/bundle/index.md:18]: Not part of this PR, but we should have a tutorial on how to setup jdk / gatling_home on windows.

File: content/tutorials/bundle/index.md [hunk 13]
```diff
@@ -2,19 +2,21 @@
...
 
-## Download the zip bundle
 [`Click here to download the latest version of the Gatling bundle`](https://gatling.io/open-source/start-testing) and extract the archive.
```
Comment (ccedric) [inline content/tutorials/bundle/index.md:20]: wrong link, the correct one is: https://gatling.io/open-source/#downloadgatling

File: content/tutorials/_includes/download_samples.md [hunk 14]
```diff
@@ -1,7 +1,6 @@
-Once you've connected to FrontLine, you'll be redirected to the simulations page.
-For this tutorial, we'll use the samples included with FrontLine.
-Click on the Download samples button.
+Once you've connected to Gatling Enterprise, you'll be redirected to the [simulations]({{< ref "../../../reference/user/simulations" >}}) page.
+We'll use the samples available by clicking on the **Sample simulations** button.
 
-{{< img src="download_samples.png" alt="Download samples" >}}
+{{< img src="download-samples.png" alt="Download samples" caption="Download samples">}}
```
Comment (Isammoc) [inline content/tutorials/_includes/download_samples.md:4]:
```suggestion
{{< img src="download-samples.png" alt="Sample simulations" caption="Sample simulations">}}
```
